### ISSUE TITLE
Install

### ISSUE THREAD

**Issue by dan1211** [doug/depthjs#8]
I have no programming experience at all but i really want to use this, is there any update on when will the simple install will be ready

**Comment by vivek-pai** [comment]
Yes, I too would like to be able to use this, but I have no idea how to set this up.

**Comment by FethrdWlf** [comment]
So, what your going to want to do is install the drivers for Kinect to your PC. Visit this page for help on that:
http://openkinect.org/wiki/Getting_Started
It's not too too hard. Then, from there I'm not sure either!....
